src/cpp/src/continuous_batching_pipeline.cpp

```diff
@@ -116,6 +122,17 @@ class ContinuousBatchingPipeline::Impl {
         return m_pipeline_metrics;
     }
 
+    std::vector<std::string> get_model_configuration() {
+        return m_model_config_namevalues;
+    }
+
+    void print_model_configuration() {
```

Comment:
You can introduce a function to cast `vector<string>` to `string`. That way every onegets a way to print it by writing `cout << cast() << '\n';`. But additionally that would allow to save the output to any other stream. File, for example.